README.md

```diff
@@ -6,11 +6,15 @@ You can use these scripts to build the .NET Core product for Windows, macOS or L
 
 ## Using the Scripts
 
-The scripts are currently support only Linux at the moment. Windows and OSX are in the pipeline.
+The scripts are supported on Windows, macOS and Linux. The scripts are based on PowerShell on Windows and Bash on macOS and Linux.
 
```

Comment:
It would be helpful if readme would indicate that Windows and macOS support is a work in progress since both still do not compile.

Reply:
Hmm, didn't realize that until now. Good point! Maybe this can be rephrased a bit:

> #### Usage
>
> The entire build consists of multiple scripts in this repo and its submodules. The main bootstrapping script are:
>
> * Windows - `build.cmd` from Command Prompt or `build.ps1` from PowerShell
>    * note that build.cmd simply invokes build.ps1
> * Unix - `build.sh`

Then a separate `Known Issues` section under `Usage` with link to https://github.com/dotnet/source-build/issues/105 as one of the most concerning issue. Something like that might work?

Reply:
From my perspective (as a user of repo) it would helpful to provide link to relevant issue directly from Usage to avoid TLDR syndrome 😄 which is quite common when user has limited time -> what is true for most of us I think.